usaspending_api/etl/management/commands/load_submission.py

```diff
@@ -514,12 +515,16 @@ def get_or_create_object_class(row_object_class, row_direct_reimbursable, logger
     return obj_class
 
 
-def get_or_create_program_activity(program_activity_code):
+def get_or_create_program_activity(row, submission_attributes):
     # We do it this way rather than .get_or_create because we do not want to
     # duplicate existing pk's with null values
-    prg_activity = RefProgramActivity.objects.filter(program_activity_code=program_activity_code).first()
-    if prg_activity is None and program_activity_code is not None:
-        prg_activity = RefProgramActivity.objects.create(program_activity_code=program_activity_code)
+    filters = {'program_activity_code': row['program_activity_code'],
+               'budget_year': fy(submission_attributes.reporting_period_start),
+               'responsible_agency_id': row['agency_identifier'],
```

Comment:
they're not always the same, but we should be using the AID since the program activities map to the AID of the TAS, not necessarily the CGAC code of the agency submitting.